presto-native-execution/presto_cpp/main/ServerOperation.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "presto_cpp/main/ServerOperation.h"
+#include "velox/common/base/Exceptions.h"
+
+namespace facebook::presto {
+ServerOperation::Target ServerOperation::targetFromString(
+    const std::string& str) {
+  auto it = kTargetLookup.find(str);
+  if (it == kTargetLookup.end()) {
+    VELOX_USER_FAIL("Unsupported server operation target '{}'", str);
+  }
+  return it->second;
+}
+
+std::string ServerOperation::targetString(ServerOperation::Target target) {
+  auto it = kReverseTargetLookup.find(target);
```

Comment:
Let's implement tagetString use a switch case as Target is well defined enum and we can easily handle error using default case? Thanks!

Reply:
I'm using a lookup/reverselookup mainly to guard the case that developers try to add something and forget to update something. I have a test guarding this. And btw just curious what do you mean by easily handling errors?